### PR TITLE
RMMIS-6451: Update EntryList component

### PR DESCRIPTION
**Pull request by kjschulz**

### CODE REVIEW

File: test/unit/entrylist-spec.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    }
+    // rerender the component
+    comp.setState();
+  };
+
+  it('renders an entrylist', () => {
+    expect(entrylist).toBeDefined();
+    expect(table.className).toEqual('table table-striped table-bordered table-hover');
+    // extra column is for actions
+    expect(columns.length).toEqual(config.columns.length + 1);
```
Comment (mallenscdhhs) [inline test/unit/entrylist-spec.js]: you also need to test that the form is in the correct state as well as the "Add" button.

**Comment by mallenscdhhs** [comment]
@kjschulz you got a test failing here

File: test/unit/entrylist-spec.js (hunk 1 above)
Comment (kjschulz) [inline test/unit/entrylist-spec.js]: Updated.

**Comment by kjschulz** [comment]
@mallenscdhhs & @traviscooperscdhhs, there seems to be a problem with the `factory-spec` tests.  I show 3 tests failing on the latest for 3.0 even if I hide all my tests, but if I hide the Factory tests, all tests including this PR's tests pass.

**Comment by michaelleeallen** [comment]
Have you pulled down the latest from release branch?

**Comment by kjschulz** [comment]
Yessir :(

John

**Comment by mallenscdhhs** [comment]
:+1:

**Pull request merged**